### PR TITLE
[FR] Update `Contests`

### PR DESCRIPTION
**Pull request by Pisapou**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Contests/fr.md [hunk 1]
```diff
@@ -186,6 +177,7 @@ Le **Fan Art Contest** est un concours de dessin dans lequel les artistes partic
...
 | [#31 (Spring 2022)](https://osu.ppy.sh/home/news/2022-04-11-place-to-belong-fanart-contest) | A Place to Belong | ::{ flag=ID }:: [Dreamxiety](https://osu.ppy.sh/users/13103233) | ::{ flag=NL }:: [utaaa](https://osu.ppy.sh/users/9315038) | ::{ flag=US }:: [Llamartist](https://osu.ppy.sh/users/14697690) | [#1](https://osu.ppy.sh/home/news/2022-05-08-aptb-fanart-results) | [#1](https://osu.ppy.sh/community/contests/142) |
+| [#32 (15th Anniversary Art Contest)](https://osu.ppy.sh/home/news/2022-08-15-15-anniversary-art-contest) | Les mascottes d'osu! qui explorent une partie de l'histoire d'osu! | ::{ flag=ID }:: [Dreamxiety](https://osu.ppy.sh/users/13103233) | ::{ flag=HK }:: [illuminae2005](https://osu.ppy.sh/users/29866964) | ::{ flag=GB }:: [urgutz](https://osu.ppy.sh/users/11731553) | [#1](https://osu.ppy.sh/home/news/2022-09-15-anniversary-art-contest-results) | [#1](https://osu.ppy.sh/community/contests/153) |
```
Comment (Walavouchey) [inline wiki/Contests/fr.md:180]: there's one other theme that's translated, maybe all of them should be?

File: wiki/Contests/fr.md [hunk 2]
```diff
@@ -239,15 +231,27 @@ L'**osu!trivium Quiz** est un concours de questions où les joueurs s'affrontent
...
+
+| Numéro du concours | Juge | ![Couronne en or][GCrown] | ![Couronne en argent][SCrown] | ![Couronne en bronze][BCrown] | Resultats |
+| :-: | :-- | :-- | :-- | :-- | :-- |
+| [#1](https://osu.ppy.sh/community/forums/topics/1624880) | ::{ flag=NL }:: [Boaz](https://osu.ppy.sh/users/13302996) | ::{ flag=JP }:: [_Rise](https://osu.ppy.sh/users/5217107) | ::{ flag=CL }:: [Metsaller](https://osu.ppy.sh/users/4364791) | ::{ flag=FI }:: [Antti](https://osu.ppy.sh/users/13281473) | [Results](https://docs.google.com/spreadsheets/d/1aR1GjF8diyjFhK_k_uOkU40zIKjaQbXhVbGDGmq5pdg/edit?usp=sharing) |
+| [#2](https://osu.ppy.sh/community/forums/topics/1639586) | ::{ flag=AR }:: [Axer](https://osu.ppy.sh/users/7299864) | ::{ flag=TH }:: [Ph0eNiiXZ](https://osu.ppy.sh/users/9463721) | ::{ flag=SG }:: [\_gt](https://osu.ppy.sh/users/8301957) | ::{ flag=US }:: [Vixlowo](https://osu.ppy.sh/users/20295114) | [Results](https://docs.google.com/spreadsheets/d/1ps3bOL2JlzwPEh0gVTLHnp7E48stwqHMH6-ljDSBJQs/edit?usp=sharing) |
```
Comment (Walavouchey) [inline wiki/Contests/fr.md:243]: guess this was missed in the english version

```suggestion
| [#1](https://osu.ppy.sh/community/forums/topics/1624880) | ::{ flag=NL }:: [Boaz](https://osu.ppy.sh/users/13302996) | ::{ flag=JP }:: [_Rise](https://osu.ppy.sh/users/5217107) | ::{ flag=CL }:: [Metsaller](https://osu.ppy.sh/users/4364791) | ::{ flag=FI }:: [Antti](https://osu.ppy.sh/users/13281473) | [#1](https://docs.google.com/spreadsheets/d/1aR1GjF8diyjFhK_k_uOkU40zIKjaQbXhVbGDGmq5pdg/edit?usp=sharing) |
| [#2](https://osu.ppy.sh/community/forums/topics/1639586) | ::{ flag=AR }:: [Axer](https://osu.ppy.sh/users/7299864) | ::{ flag=TH }:: [Ph0eNiiXZ](https://osu.ppy.sh/users/9463721) | ::{ flag=SG }:: [\_gt](https://osu.ppy.sh/users/8301957) | ::{ flag=US }:: [Vixlowo](https://osu.ppy.sh/users/20295114) | [#1](https://docs.google.com/spreadsheets/d/1ps3bOL2JlzwPEh0gVTLHnp7E48stwqHMH6-ljDSBJQs/edit?usp=sharing) |
```

File: wiki/Contests/fr.md [hunk 3]
```diff
@@ -160,32 +151,33 @@ Le **Fan Art Contest** est un concours de dessin dans lequel les artistes partic
...
-| [#27 (Spring 2021)](https://osu.ppy.sh/home/news/2021-04-23-spring-2021-fanart-contest) | A Day In The Park | ::{ flag=US }:: [Tofumang](https://osu.ppy.sh/users/4817223) | ::{ flag=MY }:: [Kyvrin](https://osu.ppy.sh/users/11589256) | ::{ flag=BG }:: [risuuna](https://osu.ppy.sh/users/7266506) | [#1](https://osu.ppy.sh/home/news/2021-05-20-spring-fanart-contest-results) |
-| [#28 (Summer 2021)](https://osu.ppy.sh/home/news/2021-07-29-summer-track-field-fanart-contest) | Track and Field | ::{ flag=HK }:: [Porukana](https://osu.ppy.sh/users/12992775) | ::{ flag=ID }:: [Ravioli-](https://osu.ppy.sh/users/8678404) | ::{ flag=US }:: [SvenJing](https://osu.ppy.sh/users/11215996) | [#1](https://osu.ppy.sh/home/news/2021-08-23-summer-tf-fanart-contest-results) | [#1](https://osu.ppy.sh/community/contests/133) |
+| [#26 (Winter 2020)](https://osu.ppy.sh/home/news/2020-09-11-winter-sports-2020-fanart-contest) | Sport d'Hiver | ::{ flag=US }:: [Tofumang](https://osu.ppy.sh/users/4817223) | ::{ flag=ID }:: [CheekyIfa](https://osu.ppy.sh/users/14406940) | ::{ flag=RU }:: [Dem4eg-](https://osu.ppy.sh/users/7206818) | [#1](https://osu.ppy.sh/home/news/2020-12-02-winter-2020-arrives) |
```
Comment (Ganondorf-1) [inline wiki/Contests/fr.md:174]:
```suggestion
| [#26 (Winter 2020)](https://osu.ppy.sh/home/news/2020-09-11-winter-sports-2020-fanart-contest) | Sports d'Hiver | ::{ flag=US }:: [Tofumang](https://osu.ppy.sh/users/4817223) | ::{ flag=ID }:: [CheekyIfa](https://osu.ppy.sh/users/14406940) | ::{ flag=RU }:: [Dem4eg-](https://osu.ppy.sh/users/7206818) | [#1](https://osu.ppy.sh/home/news/2020-12-02-winter-2020-arrives) |
```

File: wiki/Contests/fr.md [hunk 4]
```diff
@@ -239,15 +231,27 @@ L'**osu!trivium Quiz** est un concours de questions où les joueurs s'affrontent
 
 Les *concours non officiels* sont des concours organisés par les membres de la communauté.
 
-### osb! Storyboarding Contest
+### [Mapping to Pander: Taiko](M2P)
+
+**Mapping to Pander: Taiko** est une série de concours de mapping osu!taiko avec deux caractéristiques principales : il n'y a qu'un seul juge par itération, et les participants peuvent map n'importe quelle chanson.
```
Comment (Ganondorf-1) [inline wiki/Contests/fr.md:236]:
```suggestion
**Mapping to Pander: Taiko** est une série de concours de mapping osu!taiko avec deux caractéristiques principales : il n'y a qu'un seul juge par itération, et les participants peuvent mapper n'importe quelle musique.
```

File: wiki/Contests/fr.md [hunk 5]
```diff
@@ -239,15 +231,27 @@ L'**osu!trivium Quiz** est un concours de questions où les joueurs s'affrontent
...
+
+Le but de ce concours est simple : les participants doivent créer une map qui plaise le plus au juge choisi pour le concours. Le juge ne recevra aucun critère ; il lui appartiendra de noter et de classer les maps, soit en établissant ses propres critères, soit en classant les maps du meilleur au pire.
```
Comment (Ganondorf-1) [inline wiki/Contests/fr.md:238]:
```suggestion
Le but de ce concours est simple : les participants doivent créer une map qui plaise le plus au juge choisi pour le concours. Le juge ne recevra aucun critère ; il lui appartiendra de noter et de classer les beatmaps, soit en établissant ses propres critères, soit en classant les beatmaps de la meilleure à la pire.
```

File: wiki/Contests/fr.md [hunk 6]
```diff
@@ -239,15 +231,27 @@ L'**osu!trivium Quiz** est un concours de questions où les joueurs s'affrontent
...
+
+| Numéro du concours | Juge | ![Couronne en or][GCrown] | ![Couronne en argent][SCrown] | ![Couronne en bronze][BCrown] | Resultats |
```
Comment (Ganondorf-1) [inline wiki/Contests/fr.md:240]:
```suggestion
| Numéro du concours | Juge | ![Couronne en or][GCrown] | ![Couronne d'argent][SCrown] | ![Couronne de bronze][BCrown] | Résultats |
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

**Review by Ganondorf-1 — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**